ext/snmp/tests/snmp_set_string_output_format.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Function snmp_set_string_output_format
+--CREDITS--
+Larry Adams
```

Comment:
Please omit the `--CREDITS--` section for new tests (Git already tracks who wrote the test :)

Reply:
I'm good with that.  Just making it up as I go.

Reply:
Done.